pkgs/tools/networking/httperf/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, openssl }:
+
+stdenv.mkDerivation rec {
+  baseName = "httperf";
```

Comment:
`baseName` is not a required attribute of mkDerivation. If you want to use this, I think `pname` is a more common attribute name for this idea.